src/diffusers/hooks/group_offloading.py

```diff
@@ -156,27 +156,27 @@ def _pinned_memory_tensors(self):
         finally:
             pinned_dict = None
 
-    def _transfer_tensor_to_device(self, tensor, source_tensor, default_stream):
+    def _transfer_tensor_to_device(self, tensor, source_tensor):
         tensor.data = source_tensor.to(self.onload_device, non_blocking=self.non_blocking)
         if self.record_stream:
-            tensor.data.record_stream(default_stream)
+            tensor.data.record_stream(self._torch_accelerator_module.current_stream())
```

Comment:
Could you elaborate on this change? 
https://github.com/huggingface/diffusers/pull/12721/#issuecomment-3584550990 explains why it's the way it is.